calm-ingest/prototype.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
```

Comment:
This script is a bit rough around the edges (and in particular, is pulling from local cache rather than DynamoDB, but proves the idea).